src/main/java/frc/robot/subsystems/amp/AmpIO.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.subsystems.amp;
+
+import org.littletonrobotics.junction.AutoLog;
+
+import frc.robot.subsystems.shooter.ShooterIO.ShooterIOInputs;
+
+public interface AmpIO {
+
+  @AutoLog
+  public static class AmpIOInputs {
+    public double ampBarCurrentPosition = 0.0;
+    public double ampBarAppliedVolts = 0.0;
+    public double ampBarSetpoint = 0.0;
+    public double ampBarVelocity = 0.0;
+    public double ampShooterVelocity = 0.0;
+    public double ampShooterSpeedPoint = 0.0;
+    public double ampShooterVoltage = 0.0;
+
+
+
+  }/** Updates the set of loggable inputs. */
+  public default void updateInputs(AmpIOInputs inputs) {}
+
+  /** Sets the angle of the amp bar, in radians. */
+  public default void setPosition(double positionRad) {}
+
+  /** Run open loop at the specified voltage. */
+  public default void setVoltage(double volts) {}
```

Comment:
Function voltage should be specified for which component of the amp mech is being set. For example instead of set voltage you should have setPivotVoltage and setSpinnerVoltage. Same for set position, etc.